packages/fake-backends/src/FakeChatThreadClient.ts

```diff
@@ -361,6 +364,17 @@ export class FakeChatThreadClient implements IChatThreadClient {
     return pagedAsyncIterator(this.checkedGetThread().readReceipts);
   }
 
+  uploadImage(
```

Comment:
the new interface was added in latest chat sdk, so we need to update the mock as well